extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getInstartLogicBootstrap.mustache

```diff
@@ -5,14 +5,20 @@
 	], function (doc, adContext) {
 		var context = adContext.getContext();
 
-
 		if (context.opts.instartLogicRecovery) {
+			var event = doc.createEvent('Event');
+			event.initEvent('rcv.ilb', true, false);
+
+			doc.addEventListener('rcv.ilb', function () {
```

Comment:
Can we simplify this solution by simply defining a function and calling it when required?